grunt/config/replace.js

```diff
@@ -0,0 +1,56 @@
+module.exports = function (grunt, options) {
+
+  var courseDir = options.sourcedir + 'course/';
+
+  if (options.outputdir.indexOf('build/') > -1) {
+    courseDir = options.outputdir + 'course/';
+  }
+
+  var configJson = grunt.file.readJSON(courseDir + 'config.json');
+  var defaultLanguage = configJson._defaultLanguage || 'en';
+  var courseJson = grunt.file.readJSON(courseDir + defaultLanguage + '/course.json');
+
+  // Backwards compatibility for courses missing 'description'
+  if (!courseJson.hasOwnProperty('description')) {
+    courseJson.description = '';
+  }
+
+  // A shim for edge cases where xAPI has not been configured.
+  if (!configJson.hasOwnProperty('_xapi')) {
```

Comment:
Templating would have sorted this, string replacement is OK but it ain't got no logic. 
Cool, well, leave it in for the time being then. We can figure it out later. If everyone is OK with that?